plugins/rh-rbac-backend/src/service/policies-rest-api.ts

```diff
@@ -96,6 +116,40 @@ export class PolicesServer {
       response.send({ status: 'Authorized' });
     });
 
+    router.get('/plugins/policies', async (req, response) => {
+      const decision = await this.authorize(
+        this.identity,
+        req,
+        this.permissions,
+        {
+          permission: pluginPolicyEntityReadPermission,
```

Comment:
Also noted on the other file I do not think we should introduce a new permission for this

Reply:
OK.